generator/sbpg/targets/python.py

```diff
@@ -97,6 +97,15 @@ def classnameify(s):
   """
   return ''.join(w if w in ACRONYMS else w.title() for w in s.split('_'))
 
+
+def has_real_message(l):
```

Comment:
this can be re-used in the .jj2 wherever they have
```jinja2
((* for m in msgs *))
  ((*- if m.static *))
    ((*- if not m.is_real_message *))
      ...
```